tests/Integration/AnalyticsClientTest.php

```diff
@@ -61,7 +61,10 @@ public function testAbTesting()
             $cpt++;
         } while (false);
 
-        $response = $analyticsClient->addABTest($abTest);
+        $response = TestHelper::retry(function () use ($analyticsClient, $abTest) {
+            return $analyticsClient->addABTest($abTest);
+        }, 1000, 40);
```

Comment:
Shouldn't this be `100000` everywhere since we use `usleep`?
```suggestion
        }, 100000, 40);
```